packages/cadl-python/tasks.py

```diff
@@ -21,7 +22,11 @@
 init()
 
 PLUGIN_DIR = Path(os.path.dirname(__file__))
+PLUGIN = (PLUGIN_DIR / "dist/src/index.js").as_posix()
 CADL_RANCH_DIR = PLUGIN_DIR / Path("node_modules/@azure-tools/cadl-ranch-specs")
+CADL_PROJECT = {
+    "hello": "azure-hello"
```

Comment:
I think we can extend this to be for more than package name. We could do

```python
CADL_SPECIAL_FLAGS = {
  "hello": [("package-name", "azure-hello")]
}
```
Or something like that, so we have the option to specify multiple flags for each package

Reply:
updated